db/factories/Shifts/ShiftTypeFactory.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Database\Factories\Engelsystem\Models\Shifts;
+
+use Engelsystem\Models\Shifts\ShiftType;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+class ShiftTypeFactory extends Factory
+{
+    /** @var string */
+    protected $model = ShiftType::class;
```

Comment:
PHP prop types

```suggestion
    protected string $model = ShiftType::class;
```

Reply:
Sadly not possible here as it inherits the base class which has no type definition here